### PR TITLE
[PL] Fix sneaky unicode characters into proper ones

### PR DESCRIPTION
**Pull request by javnik36**
**tl;dr**: Fixes pl characters written as two unicode characters instead of single one

**Long version**: Funny issue I wouldn't find without recent move to new-ish font with PL characters. In several places I found that pl characters are still displayed in fallback font instead of pretty one. After quick investigation it looks like something in my setup caused some pl characters to be written as 2 unicode characters, e.g. `ę` (normal `e` + [U+0328](https://www.compart.com/en/unicode/U+0328)) which used near each other looks exactly the same as single character `ę` ([U+0119](https://www.compart.com/en/unicode/U+0119)). BUT! There is no U+0328 glyph in arkhamic nor in teutonic 🙃, so this probably causes font fallback.
I ended up creating script to find&replace all wrong unicode combinations of pl characters, so that's why this PR looks so massive and in the same time looks like I didn't change anything at all. Hopefully that will fix the issue.

### CODE REVIEW

**Comment by zzorba** [comment]
Isn't Unicode the best.

Thanks for taking the time to sleuth this out.
